README.md

```diff
@@ -1,5 +1,5 @@
 # LMake
-First of all, this project is currently on **heavy development**. If you are using this software, all bug reports and suggestions are welcome!
+This branch is where all the stable versions reside, if you want to check development versions go to check the 'dev' branch.
```

Comment:
This change is totally unnecessary. Readme of the dev branch is okay as is, no need to change it to stable branch readme.

Reply:
At merge this change is going to be taken out.